docs/include/content.tex

```diff
@@ -30,6 +30,34 @@ \chapter{Requirements}
             \path [connector] (result) -- (end);
         \end{tikzpicture}
 
+        \begin{tikzpicture}
+            \begin{class}[text width=8cm]{Bearing}{0,0}
+                \attribute{- Cdyn : double}
+                \attribute{- Y : double}
+                \attribute{- e : double}
+                \attribute{- xB1 : double}
+                \attribute{- xD1 : double}
+                \attribute{- xD2 : double}
+
+                
+                
+            \end{class}
+        \end{tikzpicture}
+
+        \begin{tikzpicture}
+            \begin{class}[text width=8cm]{OArrangement}{0,0}
+                \attribute{- bearingA : Bearing}
+                \attribute{- bearingB : Bearing}
+                \attribute{- Fr : double}
```

Comment:
Fr & Fa and their corresponding geometric paramters are loads on the shaft-bearing-system. May it be better to seperate them in a class "Load"?

n, the rotational speed, belongs to the load class as well.

Reply:
It seems logical, I will create the class "Load".